configs/setups/awicm3/awicm3.yaml

```
@@ -552,8 +681,16 @@ xios:
 rnfmap:
         pool_dir: ${general.pool_dir}
         runoff_maps_name: _${fesom.resolution}
-
-        include_env_from_component_files: False
+        choose_general.major_version:
+                "v3.3":
+                        # TODO: change this to a control on the patch version general.version
```

This needs to be dealt with.

---

Meaning no more major_version? Everything is choose_version?

---

I mean:
```suggestion
                        choose_general.version:
                                "v3.3.0":
                                        version: awicm-3.1
                                "*":
                                        version: v1.1
```
This is to reduce the number of choose_resolution dependencies. At the moment version depends on -> general.major_version -> fesom.version which also depends on -> general.version.

In the way I am saying there is one choose less to be resolved in the backend: general.major_version -> general.version

---

Do not commit this suggestion, I made one below equivalent to this one